tests/queries/0_stateless/03636_storage_alias_basic.sql

```diff
@@ -232,4 +233,31 @@ INSERT INTO metadata_alias VALUES (5, 'five', 'data5');
 SELECT id, value, extra FROM metadata_alias WHERE id = 5;
 
 DROP TABLE metadata_alias;
-DROP TABLE metadata_target;
\ No newline at end of file
+DROP TABLE metadata_target;
+
+SELECT 'Test alias with missing target table';
+DROP TABLE IF EXISTS missing_target_alias;
+DROP TABLE IF EXISTS temp_target;
+
+CREATE TABLE temp_target (id UInt32, value String) ENGINE = MergeTree ORDER BY id;
+INSERT INTO temp_target VALUES (1, 'data1'), (2, 'data2');
+
+CREATE TABLE missing_target_alias ENGINE = Alias('temp_target');
+SELECT * FROM missing_target_alias ORDER BY id;
+
+DROP TABLE temp_target;
+
+SELECT name, engine FROM system.tables WHERE database = currentDatabase() AND name = 'missing_target_alias';
+
+SELECT count() > 0 AS has_columns FROM system.columns WHERE database = currentDatabase();
+SELECT count() > 0 AS has_tables FROM system.tables WHERE database = currentDatabase();
```

Comment:
For which tables we have columns and which tables? Let's print them (`arraySort(groupUniqArray())`)